api/src/pcapi/models/feature.py

```diff
@@ -132,6 +132,9 @@ class FeatureToggle(enum.Enum):
     WIP_DISABLE_NOTIFY_USERS_BOOKINGS_RETRIEVED = (
         "Désactiver la notification push Batch pour les réservations sur le point d'expirer"
     )
+    WIP_DISABLE_SEND_NOTIFIACTIONS_FAVORITES_NOT_BOOKED = (
```

Comment:
```suggestion
    WIP_DISABLE_SEND_NOTIFICATIONS_FAVORITES_NOT_BOOKED = (
````

Mais je vois trois conventions de nommage en quatre FF : 
- `..._NOTIFICATION`
- `..._NOTIFY_...`
- `..._SEND_NOTIFICATIONS_...`

N'est-il pas préférable d'avoir un nommage cohérent entre tous ?

Reply:
Bien vu pour le nom
Pour le reste, je me suis un peu basé sur le nom des fonctions concernées, mais je peux changer si tu préfères